src/components/TextInput.tsx

```diff
@@ -26,6 +27,7 @@ type Props = Omit<RNTextInputProps, 'style'> & {
   onChangeText: (value: string) => void
   testID?: string
   showClearButton?: boolean
+  right?: React.ReactNode
```

Comment:
this variable name is hard to understand. 
```suggestion
  rightElement?: React.ReactNode
```

Reply:
I named it initially this way to follow the same naming as in CustomHeader:
https://github.com/valora-inc/wallet/blob/ac248834c6b5dfed860330c51bd9a8c3ac6dbaab/src/components/header/CustomHeader.tsx#L7

Renamed for clarity